### PR TITLE
Release/1.5

### PR DESCRIPTION
**Pull request by mohamed-garci**

### CODE REVIEW

File: PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/diagnosis/DiagnosisHelper.kt [hunk 1]
```diff
@@ -12,14 +12,17 @@ import android.net.Uri
 import android.os.Build
 import android.os.PowerManager
 import android.provider.Settings
-import android.support.v4.app.ActivityCompat
 import android.support.v4.app.NotificationManagerCompat
 import android.support.v4.content.ContextCompat
 import android.util.Log
 import com.drivequant.drivekit.common.ui.utils.DKReachability
+import com.drivequant.drivekit.permissionsutils.BuildConfig
```
Comment (Steven-DriveQuant) [inline PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/diagnosis/DiagnosisHelper.kt:19]: @mohamed-garci Check these 4 unused imports

File: PermissionsUtilsUI/src/main/res/values/dimen.xml [hunk 2]
```diff
@@ -2,4 +2,5 @@
 <resources>
```
[the remark this answers is not in the document]
Comment (mohamed-garci) [reply PermissionsUtilsUI/src/main/res/values/dimen.xml:2]: @Steven-DriveQuant There is a certain symmetry between the diagnosis header and the second section that can't be achieved without custom dimen

File: PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/diagnosis/DiagnosisHelper.kt (hunk 1 above)
Comment (mohamed-garci) [reply PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/diagnosis/DiagnosisHelper.kt]: @Steven-DriveQuant Cleaned

File: PermissionsUtilsUI/src/main/res/values/dimen.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Steven-DriveQuant) [reply PermissionsUtilsUI/src/main/res/values/dimen.xml:2]: @mohamed-garci Sorry I made a mistake and my review was deleted.

1/ Remove dimen.xml and the custom values in the UI module
2/ In `activity_app_diagnosis` layout :
- In the ConstraintLayout change horizontal paddings by horizontal margins and value : `dk_margin_half`
- In the summary title change custom dimen by `dk_margin_half` 
- In the summary description change `diagnosis_header_margin` by `dk_margin_half`
- In each DiagnosisItemView remove marginStart declaration

**Comment by Steven-DriveQuant** [comment]
@mohamed-garci Ok for me. Thanks for the changes
@jeremybayle-pro You can review once you are available

**Comment by mohamed-garci** [comment]
@jeremybayle-pro @Steven-DriveQuant Je pense que la beta 4.10.2 marche plutôt pas mal, vous pouvez regarder la PL quand vous aurez le temps, si vous avez des remarques je suis disponible.

File: PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/permissions/activity/LocationPermissionActivity.kt [hunk 3]
```diff
@@ -27,6 +30,12 @@ class LocationPermissionActivity : BasePermissionActivity() {
         }
     }
 
+    private fun setToolbar() {
```
Comment (jeremybayle-pro) [inline PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/permissions/activity/LocationPermissionActivity.kt:33]: @mohamed-garci mettre cette fonction dans la class BasePermissionActivity parent plutôt que de la copier-coller dans chaque activity

Comment (mohamed-garci) [reply PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/permissions/activity/LocationPermissionActivity.kt]: @jeremybayle-pro Done

File: PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/permissions/activity/BasePermissionActivity.kt [hunk 4]
```diff
@@ -12,6 +15,19 @@ open class BasePermissionActivity : RequestPermissionActivity() {
 
     private var nextPermissionViews = ArrayList<PermissionView>()
 
+    protected fun setToolbar() {
+       val titleResId = when (this) {
```
Comment (jeremybayle-pro) [inline PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/permissions/activity/BasePermissionActivity.kt:19]: Il vaut mieux passer le title en attribut de la méthode setToolbar plutôt que de throw une exception

Comment (mohamed-garci) [reply PermissionsUtilsUI/src/main/java/com/drivequant/drivekit/permissionsutils/permissions/activity/BasePermissionActivity.kt]: @jeremybayle-pro Done

**Pull request merged**